modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -0,0 +1,262 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-account.adoc
+
+
+[id="minimum-required-azure-permissions_{context}"]
+= Required Azure permissions in installer-provisioned infrastructure
+
+When you assign the `Contributor` and `User Access Administrator` roles to the service principal, you automatically grant all the required permissions. To deploy and destroy an {product-title} cluster with installer-provisioned infrastructure, the service principal requires the following permissions.
+
+Following are the permissions required for creating the {product-title} cluster on Azure public cloud. You can use the same permissions to create the private {product-title} cluster on Azure.
```

Comment:
Since network DNS zones related permissions are not required for private cluster. Maybe we can reword here.

You can use the same permissions to create the private {product-title} cluster on Azure.
=> 
You can use following permissions but remove network DNS zone related permissions to create the private {product-title} cluster on Azure.

@xenolinux @chiragkyal  any suggestion?

Reply:
Addressed - Added the note for removing the dnszones related permissions for the private cluster